api/package.json

```diff
@@ -12,6 +12,6 @@
     "date-fns": "2.25.0",
     "date-fns-tz": "1.1.6",
     "ics": "2.31.0",
-    "node-fetch": "2.6.5"
+    "node-fetch": "3.0.0"
```

Comment:
https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md

Reply:
Migrates from CommonJS modules to ESM, so incompatible with Static Web Apps Functions as it only supports Node 12 and CommonJS.
https://docs.microsoft.com/en-us/azure/static-web-apps/apis
https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module